docs/articles/sdk/using-the-sdk-to-crosscompile.rst

```diff
@@ -6,7 +6,9 @@ Using the SDK to cross compile
 Cross compiling with the help of the SDK is not much different from normal compilation. There are
 some more pitfalls, but let's get on with the happy cases for now.
 
-Sourcing Environment
+.. _sourcing-environment:
```

Comment:
I think this should be renamed as well to follow the same pattern of reflecting the heading.